db-scripts/src/main/resources/cgds.sql

```diff
@@ -865,5 +869,46 @@ CREATE TABLE `info` (
   `DB_SCHEMA_VERSION` varchar(24),
   `GENESET_VERSION` varchar(24)
 );
+
+-- --------------------------------------------------------
+CREATE TABLE `resource_definition` (
+  `RESOURCE_ID` varchar(255) NOT NULL,
+  `DISPLAY_NAME` varchar(255) NOT NULL,
+  `DESCRIPTION` varchar(2048) NOT NULL,
+  `RESOURCE_TYPE` varchar(255) NOT NULL COMMENT 'STUDY, PATIENT or SAMPLE',
+  `OPEN_BY_DEFAULT` BOOLEAN NOT NULL,
+  `PRIORITY` int(11) NOT NULL,
+  `CANCER_STUDY_ID` int(11) NOT NULL,
+  PRIMARY KEY (`RESOURCE_ID`,`CANCER_STUDY_ID`),
```

Comment:
@adamabeshouse just want to confirm that there won't be a duplicate `RESOURCE_ID` for different types within a study

Reply:
correct, that's what I think it should be. thats how it works with clinical attributes, right? (clinical attr, study) is unique? @kalletlak

Reply:
yes